build-tools/packages/build-cli/src/lib/package.ts

```diff
@@ -15,14 +15,27 @@ import { VersionSpec } from "npm-check-updates/build/src/types/VersionSpec";
 import type { Index } from "npm-check-updates/build/src/types/IndexType";
 import * as semver from "semver";
 import { isReleaseGroup, ReleaseGroup, ReleasePackage } from "../releaseGroups";
+import { DependencyUpdateType } from "./bump";
+
+/**
+ * An object that maps package names to version strings or range strings.
+ *
+ * @internal
+ */
+export interface PackageVersionMap {
+    [packageName: ReleasePackage]: string;
+}
 
 /**
  * Checks the npm registry for updates for a release group's dependencies.
  *
  * @param context - The {@link Context}.
  * @param releaseGroup - The release group to check.
```

Comment:
We should probably document that `undefined` means update everything.

Reply:
Good point.